test/integration/test_server.cpp

```diff
@@ -443,3 +443,24 @@ TEST_CASE("WriteBatch", "[integration][current]") {
     REQUIRE(res->RowCount() == 0);
   }
 }
+
+TEST_CASE("Truncate nonexistent table should succeed", "[integration]") {
+  DestinationSdkImpl service;
+
+  const std::string bad_table_name =
+      "nonexistent" + std::to_string(Catch::rngSeed());
+
+  auto token = std::getenv("motherduck_token");
+  REQUIRE(token);
+
+  ::fivetran_sdk::TruncateRequest request;
+  (*request.mutable_configuration())["motherduck_token"] = token;
+  (*request.mutable_configuration())["motherduck_database"] = "fivetran_test";
+  request.set_schema_name("some_schema");
+  request.set_table_name(bad_table_name);
+  ::fivetran_sdk::TruncateResponse response;
+  auto status = service.Truncate(nullptr, &request, &response);
+
+  INFO(status.error_message());
+  REQUIRE(status.ok());
```

Comment:
Not sure if the syntax is correct but is there a way to check the warning message?
```suggestion
  REQUIRE(status.ok());
  REQUIRE_THAT(status->warning_message(), Catch::Contains("Table <foo> not found in schema <bar>; not truncated"));
```

Reply:
yeah, that's a good idea, let met ry

Reply:
it worked!